packages/orchestrator-ui-components/src/pages/startPage/queryVariables.ts

```diff
@@ -100,12 +101,19 @@ export const taskListSummaryQueryVariables: GraphqlQueryVariables<Process> = {
     ],
 };
 
-export const productsSummaryQueryVariables: GraphqlQueryVariables<ProductsSummary> =
-    {
-        first: 1000,
-        after: 0,
-        sortBy: {
-            field: 'name',
-            order: SortOrder.ASC,
+export const productsSummaryQueryVariables: GraphqlQueryVariables<
+    ProductsSummary & Subscription
+> = {
+    first: 1000,
+    after: 0,
+    sortBy: {
+        field: 'name',
+        order: SortOrder.ASC,
+    },
+    filterBy: [
+        {
+            field: 'status',
+            value: `!${SubscriptionStatus.TERMINATED}`,
```

Comment:
I can imagine this filter happens quite a lot. Maybe have a couple of predefined field/value pairs?

Reply:
Indeed, when also used somewhere else we need to start introducing predefined filters. This is currently not the case